dev-db/redict/files/redict-7.3.0-system-jemalloc.patch

```diff
@@ -0,0 +1,168 @@
+From 5defea5b98859f4e1162b82120114e222b2a7465 Mon Sep 17 00:00:00 2001
+From: Maytham Alsudany <maytha8thedev@gmail.com>
+Date: Wed, 10 Apr 2024 01:10:49 +0300
+Subject: [PATCH 1/2] Add ability to use system jemalloc
+
+This time, this shouldn't cause CI tests to fail.
```

Comment:
Could you reference upstream PR in the patch? See https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/#patch-descriptions.

Reply:
added some comments to the top of the file